lib/braveAlerter.js

```diff
@@ -279,19 +279,28 @@ class BraveAlerter {
 
       // Store the results and idemopotently get the respondedByPhoneNumber for the session
       const changedAlertSession = new AlertSession(alertSession.sessionId, nextAlertState, fromPhoneNumber, incidentCategoryKey)
-      const respondedByPhoneNumber = await this.alertSessionChangedCallback(changedAlertSession)
+      const { respondedByPhoneNumber, replaceReturnMessageToRespondedByPhoneNumber, replaceReturnMessageToOtherResponderPhoneNumbers } =
+        await this.alertSessionChangedCallback(changedAlertSession)
 
       if (respondedByPhoneNumber === fromPhoneNumber) {
         // Send the response to the designated RespondedByPhoneNumber
         if (returnMessageToRespondedByPhoneNumber !== null) {
-          await twilioHelpers.sendTwilioMessage(respondedByPhoneNumber, toPhoneNumber, returnMessageToRespondedByPhoneNumber)
+          await twilioHelpers.sendTwilioMessage(
+            respondedByPhoneNumber,
+            toPhoneNumber,
+            replaceReturnMessageToRespondedByPhoneNumber || returnMessageToRespondedByPhoneNumber,
```

Comment:
I suggest calling these variables `replacement*` instead of `replace*`. `replace*` sounds like a verb so I would expect it to be a function or a boolean. Whereas `replacement*` sounds like the noun that it is. How do you feel about that? I know these names are already very long. Blame my Java background ;-)

Reply:
Agree with this!